tests/compiler/test_strings.py

```diff
@@ -135,3 +135,16 @@ def test_allowed_line_breaks(snippetcompiler):
 '''
     )
     compiler.do_compile()
+
+
+def test_escaping_rules_single_vs_triple_quotes(snippetcompiler):
+    snippetcompiler.setup_for_snippet(
+        r'''
+test_string_1 = """trippel hello\nworld"""
+test_string_2 = "single hello\nworld"
+std::print(test_string_1)
+std::print(test_string_2)
+'''
+    )
+    compiler.do_compile()
+
```

Comment:
This test case doesn't contain an assertion?

Reply:
Yes, this is just a visual test to check that the problem mentioned in the [issue](https://github.com/inmanta/inmanta-core/issues/2582) is solved. I will mention it in the docstring and the name of the test.

Reply:
Like that the test case is not really useful. Nobody is ever going to have a look at this output after this change is merge and we will never notice if a change breaks this functionality. Next to that, it's not really difficult to add an assertion, so let's add it.

Reply:
Of course! I was missing the point :+1: